backend/core/models.py

```diff
@@ -9,11 +9,12 @@
 # import re
 
 # from rest_framework.generics import RetrieveAPIView
+import git_real
```

Comment:
what is this import for? You just import git_real later on.

Can you please tidy up your imports?